dlc-node/src/main/scala/org/bitcoins/dlc/node/DLCServer.scala

```diff
@@ -49,6 +49,11 @@ class DLCServer(
                                    dataHandlerFactory)))
   }
 
+  override def postStop(): Unit = {
+    super.postStop()
+    socket ! DLCServer.Disconnect
```

Comment:
So the `socket` will not understand `DLCServer.Disconnect`. You need to send `Tcp.Unbind` here I believe as this is the underlying IO actor that akka manages itself. Only `DLCServer` knows what to do with the `DLCServer.Disconnect` method